src/rust/src/lib.rs

```diff
@@ -23,7 +23,7 @@ fn encode_vectorized_(what: Either<Strings, List>, engine: Robj) -> Strings {
     let eng: ExternalPtr<GeneralPurpose> = engine.try_into().unwrap();
     match what {
         Either::Left(s) => s
-            .into_iter()
+            .iter()
```

Comment:
can we revert these changes? It's resulting in a 5ms / 12 iter/sec speed reduction.

Reply:
Good you benchmarked it. I am very surprised. Iter should be more efficient. However, the optimizer is king here..

Reply:
Done, I thought `clippy` would get the best thing but apparently not

Reply:
Wait? Does this mean I have one over clippy for once in my life....?

Reply:
Yes this was done in [634ac3e](https://github.com/extendr/b64/pull/2/commits/634ac3e8b251a8bb792360db41da0bd405eb999e), that only includes changes of `cargo clippy --fix`